app/controllers/user_controller.js

```diff
@@ -19,14 +19,40 @@ exports.signUp = (req, res, next) => {
     });
 };
 
-exports.signIn = function(req, res, next) {
+exports.signIn = (req, res, next) => {
   const { email, password } = req.body;
   return User.findOne({ where: { email } }).then(user => {
     if (!user) next(error.unAuthorizedError('Invalid credentials'));
     if (!user.validatePassword(password)) next(error.unAuthorizedError('Invalid credentials'));
     if (user && user.validatePassword(password)) {
       const token = jwt.sign({ id: user._id }, config.common.session.secret);
-      res.status(200).send({ auth: true, token });
+      res
+        .status(200)
+        .cookie('x-access-token', token)
+        .send({ auth: true });
     }
   });
 };
+
+exports.getAllUsers = (req, res, next) => {
+  const limit = 5;
+  const page = req.query.page || 1;
+  User.findAndCountAll()
```

Comment:
if you are going to do this, just filter them manually  so you dont look them up twice in the db